### ISSUE TITLE
3D transform doesn't work after element is detached / reattached?

### ISSUE THREAD

**Issue by matthew-dean** [rstacruz/jquery.transit#136]
I'm using jQuery.detach() on elements. The 3D rotation works the first time, but, after i've detached and reattached an element, the rotation is only on the 2D plane, even if I've "reset" any inline styles (removed the style attribute from the selector before animating).
